aten/src/ATen/native/native_functions.yaml

```diff
@@ -3260,15 +3260,27 @@
 
 - func: adaptive_avg_pool2d_out(Tensor output, Tensor self, IntList[2] output_size) -> Tensor
   python_module: nn
+  dispatch:
+    CPU: AdaptiveAveragePooling2d_out_cpu
```

Comment:
nit: I would just name these the same as the actual function, e.g. `adaptive_avg_pool2d_out_cpu`, instead of `AdaptiveAveragePooling2d_out_cpu` -- that's the old TH naming, no need you need to preserve it.